src/xmlsec/__init__.py

```diff
@@ -38,7 +39,7 @@ class Config(object):
     default_signature_alg = pyconfig.setting("xmlsec.default_signature_alg", constants.ALGORITHM_SIGNATURE_RSA_SHA256)
     default_digest_alg = pyconfig.setting("xmlsec.default_digest_alg", constants.ALGORITHM_DIGEST_SHA256)
     default_c14n_alg = pyconfig.setting("xmlsec.default_c14n_alg", constants.TRANSFORM_C14N_EXCLUSIVE)
-    debug_write_to_files = pyconfig.setting("xmlsec.config.debug_write_to_files", False)
+    debug_write_to_files = pyconfig.setting("xmlsec.config.debug_write_to_files", True)
```

Comment:
Nice work! Probably don't want to change this default from False to True though?

Reply:
Very true, just for get coverage when running tests.